CONTRIBUTING.md

```diff
@@ -169,10 +169,20 @@ dependency_overrides:
     path: relative/path/to/devtools/packages/devtools_shared
 ```
 
-Then you can run DevTools with the server by running the following from the top-level `devtools` directory:
-```
-dt serve
-```
+Then you can run DevTools with the server by running the following from anywhere under the `devtools/` directory:
```

Comment:
these look like changes from another PR. Can you merge with master so that these changes are removed from the diff?